doc/source/myrocks/variables.rst

```diff
@@ -59,6 +59,10 @@ Also, all variables can exist in one or both of the following scopes:
      - Yes
      - No
      - Global
+   * - :variable:`rocksdb_alter_column_default_inplace`
```

Comment:
Please, mind that this two variables must go to 5.7. And just `rocksdb_enable_pipelined_write` to 8.0

Reply:
Discussed in slack.